.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
+  # Pre-reqs for building omsagent
+  - sudo apt-get install -qq pkg-config make ruby bison g++ rpm librpm-dev libpam0g-dev libssl-dev libmysqlclient-dev
+  - rvm use system                 # Use system ruby instead of ruby provided by rvm
+  - sed -i 's/--no-ri//' ~/.gemrc  # ri data from installed gems is required for our build
+
+# Travis-CI automatically clones the OMS-Agent-for-Linux repo since this is where this .travis.yml file is located
+# We will ignore this clone and clone the superproject Build-OMS-Agent-for-Linux as follows
+
+install:
+  - cd ../..
+  - git clone https://github.com/sugr4/Build-OMS-Agent-for-Linux.git sugr4/Build-OMS-Agent-for-Linux
```

Comment:
Why isn't it https://github.com/Microsoft/Build-OMS-Agent-for-Linux.git ? (i.e. why is your GitHub username in it)